src/librustdoc/clean/inline.rs

```diff
@@ -395,7 +395,14 @@ crate fn build_impl(
             impl_
                 .items
                 .iter()
-                .map(|item| tcx.hir().impl_item(item.id).clean(cx))
+                .filter_map(|item_ref| {
+                    let item = tcx.hir().impl_item(item_ref.id);
+                    if tcx.has_attr(item.def_id.to_def_id(), sym::hidden) {
+                        Some(item.clean(cx))
```

Comment:
Isn't this the other way around? :thinking:

Reply:
You're right, lol
What's funny is that the change was working well and what's weird is that the change after making it inverted keeps working well...

Reply:
Maybe it's a compilation issue? I recently had a problem where my `rustdoc` changes weren't being reflected, because I was using a wrong build command I think.